src/settings/JobProfiles/JobProfilesContainer/JobProfilesContainer.js

```diff
@@ -38,6 +38,8 @@ const JobProfilesContainer = ({
   const location = useLocation();
   const stripes = useStripes();
   const intl = useIntl();
+  const hasViewPerms = stripes.hasPerm('settings.data-export.view') && !stripes.hasPerm('module.data-export.enabled');
+  const lastMenu = hasViewPerms ? (<></>) : '';
```

Comment:
Why do we need this menu if it's empty on both cases?

Reply:
Because we pass it in `stripes-data-transfer-components ``SearchAndSortPane` where we have boolean check  ```
lastMenu={lastMenu || (
          <Button
            buttonStyle="primary paneHeaderNewButton"
            data-test-new-button
            marginBottom0
            to={`${match.path}/create${location.search}`}
          >
            <FormattedMessage id="stripes-data-transfer-components.new" />
          </Button>
        )}
      />
```
I want to use default button if we have edit permission. But if now just not show anything. So I decided to pass fragment since it's not false boolean